plugins/georeplication/rest.go

```diff
@@ -1110,8 +1097,7 @@ func georepSSHKeyPushHandler(w http.ResponseWriter, r *http.Request) {
 
 	err = txn.Do()
 	if err != nil {
-		logger.WithFields(log.Fields{
-			"error":   err.Error(),
+		logger.WithError(err).WithFields(log.Fields{
```

Comment:
Use `WithField` if it's only one field. Here and everywhere else.

Reply:
am using ```WithFields``` if I have multiple fields to log.
am not getting where I have used ```WithFields``` if I have one field to log.